src/providers/completionProvider.ts

```diff
@@ -342,7 +345,7 @@ export async function doCompletionResolve(
 
       const insertName = useFqcn ? completionItem.data.moduleFqcn : name;
       const insertText = completionItem.data.atEndOfLine
-        ? `${insertName}:`
+        ? `${insertName}:\r\t\t`
```

Comment:
This might brake in the future based on editor configuration, as it might be configured to auto-indent. still, no idea on how to do this better, just an observation.